src/Chapter2.hs

```diff
@@ -781,14 +859,24 @@ Let's now try to eta-reduce some of the functions and ensure that we
 mastered the skill of eta-reducing.
 -}
 divideTenBy :: Int -> Int
-divideTenBy x = div 10 x
+divideTenBy = div 10
 
 -- TODO: type ;)
+listElementsLessThan :: Int -> [Int] -> [Int]
 listElementsLessThan x l = filter (< x) l
 
--- Can you eta-reduce this one???
+-- Can you eta-reduce this one??? 
+{- NO 
+Why? I am guessing that if there are more than one parameters, the order of parameters matters.
+For example, zipWith (-) a b  VS zipWith (-) b a 
```

Comment:
Yes, but they would be applied in the same order as they come, so no need to worry about that. You can think of it as 2-step eta reduction. First, you remove the last argument and get:

```haskell
pairMul xs = zipWith (*) xs
```

But you immediately see that the above function could be eta-reduced itself:

```haskell
pairMul = zipWith (*)
```

Hope this makes sense. Let me know if you want to discuss eta-reduction more! 🙂